ui/webui/test/README.rst

```diff
@@ -243,3 +266,58 @@ development environment that's independent from the host::
 
     toolbox create --image quay.io/cockpit/tasks -c cockpit
     toolbox enter cockpit
+
+Web UI End to end tests
+=======================
+
+End to end tests are stored in ``./ui/webui/test/integration``
+together with tplib test cases and test plans (those are only needed when running
+the tests using Permian).
+
+End to end tests use one more level of abstraction, class `IntegrationTest <https://github.com/rhinstaller/anaconda/blob/anaconda-39.16-1/ui/webui/test/helpers/integration.py#L38>`_.
+This class handles flow through all the required installation steps with default
+options. So when writing new test you only have to use this class as parent and
+extend it or reimplement the functions that are important for the test case.
+
+End to end tests examples
+--------------------------
+
+There are three test cases in the anaconda repository that can be used as examples.
+
+**Default**
+
+Performs default installation.
+Test script ``default.py``, test case file ``default.tc.yaml``.
+
+**Storage encryption**
+
+Makes changes, compared to default installation, only in the storage section of
+the installation wizard and runs some commands before the system is rebooted.
```

Comment:
Is maybe 'It makes changes in the storage section of the installation wizard.' enough? 

I really don't know anything about e2e tests in webui, and this extra information about 'executed commands' brings me also no new knowledge.

Reply:
As I expect that all 2e2 tests do post install checks.